portal-frontend/src/app/shared/soil-table/soil-table.component.html

```diff
@@ -50,6 +51,7 @@
             separatorLimit="9999999999"
             matInput
             formControlName="volume2"
+            onfocus="this.select()"
```

Comment:
I wonder if this will cause validation to show up as soon as user clicks on the fields?